examples/next-login/src/pages/home/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react'
+import LogIn from './login'
+import SignIn from './signin'
+import { SwitchButton } from '@ultraviolet/ui'
+import { css } from '@emotion/react'
+const swichButtonStyle = css`
+  margin-left: 43%;
+`
+const Home = () => {
+  const [tab, setTab] = useState('login')
+  return (
+    <>
+      <div className="switch-button" css={swichButtonStyle}>
+        <SwitchButton
+          name="switch button"
+          leftButton={{
+            label: 'Log In',
+            value: 'login',
+          }}
+          rightButton={{
+            label: 'Sign In',
+            value: 'signin',
+          }}
+          value="login"
+          onChange={e => setTab((e.currentTarget as HTMLInputElement).value)}
+        />
+      </div>
+      {tab === 'login' ? <LogIn /> : <SignIn />}
+    </>
+  )
+}
```

Comment:
Here you can use `styled` instead of `css` and `className`, we tend to avoid using them.

We also try to avoid using one letter for variables instead of `onChange={e => e)` we would do `onChange={event => event)` it increase the understanding of what we are using. 

```suggestion
const StyledDiv = styled.div`
  margin-left: 43%;
`

const Home = () => {
  const [tab, setTab] = useState('login')
  return (
    <>
      <StyledDiv className="switch-button" css={swichButtonStyle}>
        <SwitchButton
          name="switch button"
          leftButton={{
            label: 'Log In',
            value: 'login',
          }}
          rightButton={{
            label: 'Sign In',
            value: 'signin',
          }}
          value="login"
          onChange={event => setTab((event.currentTarget as HTMLInputElement).value)}
        />
      </StyledDiv>
      {tab === 'login' ? <LogIn /> : <SignIn />}
    </>
  )
}
```